frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -148,6 +182,18 @@ export class ClusterOverview extends React.Component {
     }));
   }
 
+  _setCapacityData(key, dataKey, response) {
```

Comment:
we could move this logic to new component `ClusterCapacity` in web-ui-components and store only results of prometheus queries here. Same as done for `ClusterDetails` -> `Details` components

Reply:
To be honest, I prefer keeping the queries and parsing their result co-located. 
We pass down raw values, and consistently with the `Details` specific processing (means interpretation of the numbers) is handled with the component hierarchy

Reply:
I think the selection logic and the parsing could be reused between components and would be beneficial to move it to selectors.

Reply:
\* web-ui components selectors

Reply:
i still think that this should be moved to web-ui-components but since we want to do a build today Im good with a followup

Reply:
ok we can do it later